dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6
+
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir /parking_manager
+
+WORKDIR /parking_manager
+
+ADD . /parking_manager/
+
+RUN pip install -r requirements.txt
```

Comment:
Since it's a library that can be installed isn't it better to `pip install .` ?

Reply:
of course! I changed it